service_invocation/javascript/http/README.md

```diff
@@ -0,0 +1,66 @@
+# Dapr Service Invocation
+
+In this quickstart, you'll create a checkout service and a order processor service to demonstrate how Dapr enables a service invocation pattern. 
```

Comment:
Consider being a little more descriptive with the example. 

"The checkout service will use Dapr's http proxying capability to invoke a method in the order processing service." or something along those lines.